pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -433,6 +460,59 @@ contract LinearPool is BasePool, IGeneralPool, LinearMath, IRateProvider {
         return (balances[mainIndex] + balances[wrappedIndex]).divUp(_MAX_TOKEN_BALANCE - balances[bptIndex]);
     }
 
+    function getWrappedTokenRateProvider() public view returns (IRateProvider) {
+        return _wrappedTokenRateProvider;
+    }
+
+    function getWrappedTokenRateCache()
+        external
+        view
+        returns (
+            uint256 rate,
+            uint256 duration,
+            uint256 expires
+        )
+    {
+        rate = _wrappedTokenRateCache.getValue();
+        (duration, expires) = _wrappedTokenRateCache.getTimestamps();
+    }
+
+    function setWrappedTokenRateCacheDuration(uint256 duration) external authenticate {
```

Comment:
Do we want the owner to manage all these rate authenticate functions? In that case we should add it to `_isOwnerOnlyAction`

Reply:
I followed the same strategy we did for the meta stable pool, but let's discuss who should be able to access it